test/sanity/validate-modules/ignore.txt

```diff
@@ -617,6 +617,7 @@ lib/ansible/modules/files/synchronize.py E325
 lib/ansible/modules/files/synchronize.py E327
 lib/ansible/modules/files/unarchive.py E323
 lib/ansible/modules/identity/cyberark/cyberark_user.py E324
+lib/ansible/modules/identity/ipa/ipa_config.py E325
```

Comment:
New ignores should not be added. The underlying issues should be fixed in the code instead. If the issues are in common code, such as module_utils, then those issues should be fixed first in a separate PR.

Reply:
Thanks @mattclay! I'll open up a new PR for fix all of the ipa_ issues, and then rebase this one to fix it.

Reply:
I just opened https://github.com/ansible/ansible/pull/42282, so I'll wait to get that merged and then I'll rebase this PR with the appropriate changes.